src/NUnitEngine/nunit-agent/Program.cs

```diff
@@ -133,7 +125,8 @@ public static int Main(string[] args)
             log.Info("Initializing Services");
             engine.Initialize();
 
-            Channel = ServerUtilities.GetTcpChannel();
+            // Owns the channel used for communications with the agency and with clients
+            var testAgencyServer = engine.Services.GetService<TestAgency>();
```

Comment:
We have been starting two separate channels for no good reason which caused no small amount of confusion when prototyping the fix at 1 am last night. =D